api/src/test/java/org/openmrs/api/context/DaemonTest.java

```diff
@@ -22,23 +22,41 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.Test;
 import org.openmrs.Person;
 import org.openmrs.PersonName;
 import org.openmrs.User;
 import org.openmrs.api.APIAuthenticationException;
 import org.openmrs.api.APIException;
+import org.openmrs.api.UserService;
 import org.openmrs.scheduler.Task;
 import org.openmrs.scheduler.tasks.AbstractTask;
 import org.openmrs.scheduler.tasks.HelloWorldTask;
 import org.openmrs.scheduler.timer.TimerSchedulerTask;
 import org.openmrs.test.jupiter.BaseContextSensitiveTest;
+import org.openmrs.util.PrivilegeConstants;
 
 /**
  * Tests the methods on the {@link Daemon} class
  */
 public class DaemonTest extends BaseContextSensitiveTest {
 	
+	/**
+	 * Methods in this class might create a new user, so delete that user after this whole junit class is done.
+	 */
+	@AfterAll
+	public static void deleteUserAfterThisTest() {
+		UserService userService = Context.getUserService();
+		
+		Context.addProxyPrivilege(PrivilegeConstants.GET_USERS);
+		Context.addProxyPrivilege(PrivilegeConstants.PURGE_USERS);
+		
+		userService.purgeUser(userService.getUserByUsername("jdoe"));
+		
+		Context.removeProxyPrivilege(PrivilegeConstants.GET_USERS);
+		Context.removeProxyPrivilege(PrivilegeConstants.PURGE_USERS);
+	}
```

Comment:
@dkayiwa these changes fix the build failures but they are not related to this ticket, should I create another PR for them?!

Reply:
Does this mean that the latest upstream openmrs-core does not compile successfully?

Reply:
That is not true, according to this green CI builld: https://ci.openmrs.org/browse/TRUNK-MASTER/latest

Reply:
Oh, okay, my apologies then🙂

Reply:
So can we remove the changes which are not related to the ticket at hand?

Reply:
@dkayiwa removing these changes will lead to build failures

Reply:
Are the build failures related to this ticket?

Reply:
no they're not, okay, removing them